### ISSUE TITLE
Use issue tracker?

### ISSUE THREAD

**Issue by eriksvedang** [catnipped/WelcomeToFever#1]
Perhaps the project should use the issue tracker instead of a TODO-list? Then people can create new issues and you can assign them to different people!

**Comment by catnipped** [comment]
Good idea! I'll set up something like that once the project is big enough. Still think a main roadmap-thing to-do could be useful?
